cuckoo/data-private/cwd/conf/qemu.conf

```diff
@@ -19,7 +19,7 @@ label = {{ qemu[machine].label }}
 # image path
 image = {{ qemu[machine].image }}
 
-# vm arch (mips/mipsel/arm/x64/x86)
+# vm arch (sh4/sparc/sparc64/powerpc/powerpc64/mips/mipsel/arm/x64/x86)
```

Comment:
I think x86/x64/arm take precedence over the others, so please sort them accordingly.